cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2930,80 +2930,82 @@ action.cyences_send_email_action = 1
-action.cyences_notable_event_action.system_compromised_drilldown = (`cs_o365` (sourcetype="ms:o365:reporting:messagetrace" OR sourcetype="o365:reporting:messagetrace")) OR (`cs_gws` sourcetype="gws:gmail") | `cs_email_search_suspicious_subject_or_attachment` | search SenderAddress=$row.SenderAddress$ RecipientAddress=$row.RecipientAddress$ Subject=$row.Subject$
-action.cyences_send_email_action = 1
-action.cyences_notable_event_action.products = Office 365, Google Workspace
-
-
-[Email - With Known Abuse Web Service Link]
-disabled = 1
-enableSched = 1
-alert.track = 1
-alert.severity = 4
-alert.suppress = 0
-counttype = number of events
-quantity = 0
-relation = greater than
-cron_schedule = 57 * * * *
-description = This alert will trigger mail contains Known Abuse Web Service Link. \
-NOTE: cs_email_user_domain macro must be configured with the domain name in order to use this alert. \
-Data Collection - Gmail data (Splunk Add-on for Google Workspace). 
-dispatch.earliest_time = -1d@h
-dispatch.latest_time = now
-display.general.type = statistics
-display.page.search.tab = statistics
-display.page.search.mode = fast
-request.ui_dispatch_app = cyences_app_for_splunk
-request.ui_dispatch_view = search
-search = `cs_gws` sourcetype="gws:gmail" _index_earliest=-61m@m _index_latest=-1m@m \
-| `cs_email_search_known_abuse_web_service_link` \
-| search NOT src_user_domain IN `cs_email_user_domain` AND recipient_domain IN `cs_email_user_domain` \
-| stats count, min(_time) as firstTime, max(_time) as lastTime, values(link_domain) as link_domain by SenderAddress, RecipientAddress, Subject, is_spam \
-| eval cyences_severity = "low" \
-| `cs_human_readable_time_format(firstTime)` \
-| `cs_human_readable_time_format(lastTime)` \
-| sort - count \
-| `cs_email_with_known_abuse_web_service_link_filter`
-action.cyences_notable_event_action = 1
-action.cyences_notable_event_action.param.filter_macro_name = cs_email_with_known_abuse_web_service_link_filter
-action.cyences_notable_event_action.contributing_events = `cs_gws` sourcetype="gws:gmail" | `cs_email_search_known_abuse_web_service_link` | search NOT src_user_domain IN `cs_email_user_domain` AND recipient_domain IN `cs_email_user_domain`
-action.cyences_notable_event_action.system_compromised_search = | stats count by SenderAddress, RecipientAddress, Subject, is_spam
-action.cyences_notable_event_action.system_compromised_drilldown = `cs_gws` sourcetype="gws:gmail" | `cs_email_search_known_abuse_web_service_link` | search NOT src_user_domain IN `cs_email_user_domain` AND recipient_domain IN `cs_email_user_domain` | search SenderAddress=$row.SenderAddress$ RecipientAddress=$row.RecipientAddress$ Subject=$row.Subject$ is_spam=$row.is_spam$
-action.cyences_send_email_action = 1
-action.cyences_notable_event_action.products = Google Workspace
+# Commented out this alert and keep it here for future reference because currently, it's not accurate.
```

Comment:
Add reasoning on to why it is inaccurate.

Not sure if it is good idea to keep it commented.

Also, We need to remove relavent lookups and macros, etc.

Reply:
I'm thinking to comment out macro and lookup definition. & keep lookup csv file as .bak for future reference.

what do you say?